conf/application.conf

```diff
@@ -195,6 +195,13 @@ business-tax-account {
   homeUrl = "/business-account"
   vatSummaryHost = "http://localhost:9152"
   vatSummaryUrl = "/vat-through-software/vat-overview"
+  messagesUrl = "/messages"
+  manageAccountUrl = "/manage-account"
+}
+
+help-and-contact-frontend {
```

Comment:
Could you also create an `app-config-base` PR to override the `host` here to a blank string? See our other vat-summary-frontend config in that repo for an example.